### ISSUE TITLE
[DGSM 1.0.1.322] [OS: Windows 11] [Version: U53-642695-SC] Game crash at duplicant configuration

### ISSUE THREAD

**Issue by thomasmitt** [O-n-y/oxy.mods#184]
Missing Field Exception > Could not find field in class Ony.OxygenNotIncluded.DGSM2.CharacterHelper:

How to reproduce: Open the game and click to configure duplicant traits.

```
[Exception]: MissingFieldException: Field not found: string .MinionConfig.MINION_BASE_TRAIT_ID Due to: Could not find field in class
Ony.OxygenNotIncluded.DGSM2.CharacterHelper+<>c.<OnEditDuplicant>b__14_4 (TUNING.DUPLICANTSTATS+TraitVal a) (at <87845a123a2b4a5996192071db458a17>:0)
System.Linq.Enumerable+WhereSelectListIterator`2[TSource,TResult].ToList () (at <a314714511a14f84b853c03efd8682b8>:0)
System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) (at <a314714511a14f84b853c03efd8682b8>:0)
Ony.OxygenNotIncluded.DGSM2.CharacterHelper.OnEditDuplicant () (at <87845a123a2b4a5996192071db458a17>:0)
Ony.OxygenNotIncluded.DGSM2.EditableTitleBar_OnSpawn+<>c__DisplayClass0_0.<Postfix>b__0 () (at <87845a123a2b4a5996192071db458a17>:0)
KButton.SignalClick (KKeyCode btn) (at <199beda79a0545e18356f69e28ee38c2>:0)
KButton.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <199beda79a0545e18356f69e28ee38c2>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <6405a4abeb6c490388904d755e3f6782>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <6405a4abeb6c490388904d755e3f6782>:0)
UnityEngine.EventSystems.EventSystem:Update()

  [Exception]: MissingFieldException: Field not found: string .MinionConfig.MINION_BASE_TRAIT_ID Due to: Could not find field in class
Ony.OxygenNotIncluded.DGSM2.CharacterHelper+<>c.<OnEditDuplicant>b__14_4 (TUNING.DUPLICANTSTATS+TraitVal a) (at <87845a123a2b4a5996192071db458a17>:0)
System.Linq.Enumerable+WhereSelectListIterator`2[TSource,TResult].ToList () (at <a314714511a14f84b853c03efd8682b8>:0)
System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) (at <a314714511a14f84b853c03efd8682b8>:0)
Ony.OxygenNotIncluded.DGSM2.CharacterHelper.OnEditDuplicant () (at <87845a123a2b4a5996192071db458a17>:0)
Ony.OxygenNotIncluded.DGSM2.EditableTitleBar_OnSpawn+<>c__DisplayClass0_0.<Postfix>b__0 () (at <87845a123a2b4a5996192071db458a17>:0)
KButton.SignalClick (KKeyCode btn) (at <199beda79a0545e18356f69e28ee38c2>:0)
KButton.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <199beda79a0545e18356f69e28ee38c2>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <6405a4abeb6c490388904d755e3f6782>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <6405a4abeb6c490388904d755e3f6782>:0)
UnityEngine.EventSystems.EventSystem:Update()
```

**Comment by O-n-y** [comment]
DGSM is on update, currently in BETA testing, Release version will come to Steam soon
Join Discord for all news: https://discord.gg/hhetzbVyAW

**Comment by O-n-y** [comment]
was fixed in  DGSM   3.7.3.450
